workspace/scss/_responsive.scss

```diff
@@ -308,7 +308,7 @@ $screen-xl-min: 1439px;
 				position: absolute;
 			}
 			input, input:focus {
-				padding-left: 75px;
```

Comment:
I think that it depends on the translation of the word "search" .
For me in french "search" is translate to "Recherche".
For the keyword "test", I see "st" with 65px, "est" + a part of the "t" for 75px, and it's correct with 80px.